project/Build.scala

```diff
@@ -118,9 +118,9 @@ object Build {
   val mimaPreviousLTSDottyVersion = "3.3.0"
 
   /** Version of Scala CLI to download */
-  val scalaCliLauncherVersion = "1.3.2"
+  val scalaCliLauncherVersion = "1.4.0"
   /** Version of Scala CLI to download (on Windows - last known validated version) */
-  val scalaCliLauncherVersionWindows = "1.3.2"
+  val scalaCliLauncherVersionWindows = "1.4.0"
```

Comment:
Since this is brand new, we should come up with the procedure when updating `scala-cli`. 1.4.0 was just released today and is not validated by windows... but it is a change in minor and it is weird to have different `scala-cli` versions for the same scala release...

Reply:
I know but it weird to have different minor versions across different platforms for the same release of `scala`

Reply:
Normally, we should keep it until 1.4.0 is at least 30 days old.
However, might still be tempting to get this in to have the fix from https://github.com/VirtusLab/scala-cli/pull/2957...

Reply:
but the decision was made

Reply:
Yes, I know. 
Let's hold our horses until we decide on a timeline for this.

Reply:
Do we have an ETA for 3.5.0-RC3 ?